searx/templates/simple/result_templates/images.html

```diff
@@ -11,7 +11,7 @@
                         {%- if result.thumbnail_src -%}
                                 <img src="" data-src="{{ image_proxify(result.img_src) }}" alt="{{ result.title|striptags }}">
                         {%- else -%}
-                                <img src="{{ image_proxify(result.img_src) }}" alt="{{ result.title|striptags }}" loading="lazy">
+                                <img src="{{ image_proxify(result.img_src) }}" alt="{{ result.title|striptags }}" loading="lazy" width="200" height="200">
```

Comment:
The detail is hidden by default, so the width & height can be skipped here.

Reply:
Haven't tested the behavior of initial hidden images & lazy loading, but I recommend to set an initial `width` and `height` whenever `loading="lazy"` is set.

Reply:
After some digging:
* even without `loading="lazy"`, the browser won't the image (at least FF and Chrome).
* the block https://github.com/searxng/searxng/blob/5307fd841d11d0bead4ef07147b46e584b3b1fe4/searx/templates/simple/result_templates/images.html#L10-L16 can be replaced by
```jinja2
                <a class="result-images-source" {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} href="{{ result.img_src }}">
                        <img src="data:," data-src="{{ image_proxify(result.img_src) }}" alt="{{ result.title|striptags }}">
                </a>{{- "" -}}
```

Note: the empty src is replaced by `data:,` to avoid additional HTTP request (it is safety net here: it actually doesn't happen because [the src attribute is set before the image is shown](https://github.com/searxng/searxng/blob/5307fd841d11d0bead4ef07147b46e584b3b1fe4/searx/static/themes/simple/src/js/main/results.js#L57-L65)):
* https://humanwhocodes.com/blog/2009/11/30/empty-image-src-can-destroy-your-site/
* https://stackoverflow.com/questions/5775469/whats-the-valid-way-to-include-an-image-with-no-src

---

The change can be either another PR, or in this one because it all about the lazy loading after all.

Reply:
> * even without `loading="lazy"`, the browser won't the image (at least FF and Chrome).
>
> * the block ... can be replaced by
> ...
> The change can be either another PR

I have a different opinion for two reasons:

1. I think the if clause in the template is more robust and better tor read & undestand
2. IMO it is not an issue to set lazy=lodaing (including with & hight) in the template, no matter what CSS will hide or not.